src/services/atomic-swap.service.js

```diff
@@ -74,6 +74,26 @@ const serviceCreate = async (walletId, partialTx, password) => {
   return { proposalId: id };
 };
 
+/**
+ * Creates the proposal on the Atomic Swap Service, handling errors that may occur on the process
+ * @param updateParams Parameters related to the proposal itself
+ * @param {string} updateParams.partialTx Serialized PartialTx
+ * @param {string} updateParams.proposalId Proposal identifier
+ * @param {string} updateParams.password Proposal password on the Atomic Swap Service
+ * @param {number} updateParams.version Proposal version on the Atomic Swap Service
+ * @param {string} [updateParams.signatures] Proposal signatures, if any
+ * @returns {Promise<{ success: boolean }>} Returns if the operation was successful
+ */
+const serviceUpdate = async updateParams => {
```

Comment:
This is the same method name that you have in the other PR, right? And the other one has a different implementation, with more validations in the beginning.